content/python/metric_tracking_object.py.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Working with Metric_Tracking_Object.py"
```

Comment:
Drop the `.py` from the Title. Examples that have that are mostly from importing them en-masse from the existing examples repo.